generators/client/templates/react/package.json.ejs

```diff
@@ -169,7 +169,7 @@ limitations under the License.
     <%_ if (cypressTests) { _%>
     "cypress": "VERSION_MANAGED_BY_CLIENT_COMMON",
     <%_ } _%>
-    "typescript": "3.8.3",
+    "typescript": "4.0.2",
```

Comment:
@qmonmert Maybe we should start using the package.json file and define here VERSION_MANAGED_BY_CLIENT_REACT until we update it also to vue and put it on VERSION_MANAGED_BY_CLIENT_COMMON?

Reply:
You were faster. =)

Reply:
thanks guys, done ✅